Example/e2e/examplesTests/bottomTabs.e2e.ts

```diff
@@ -2,7 +2,7 @@ import { device, expect, element, by } from 'detox';
 
 describe('Bottom tabs and native stack', () => {
   beforeEach(async () => {
-    await device.reloadReactNative();
+    await device.launchApp({ newInstance: true });
```

Comment:
I have a question here. Why do we relaunch application instead of reloading RN? This was done mostly to save CI time IIRC, so it is a subject to be changed, but I need reasoning here.